### PR TITLE
Added changelog and update cron.yaml with retry config

### PR DESCRIPTION
**Pull request by jonasfj**
/CC @sigurdm, have a look a the changelog format..

### CODE REVIEW

File: cron.yaml [hunk 1]
```diff
@@ -3,3 +3,7 @@ cron:
   schedule: every tuesday 02:00
   url: /cron/datastore-backup
   target: default  # frontend
+  retry_parameters:
+    min_backoff_seconds: 3600 # wait at-least an hour before retrying
+    max_backoff_seconds: 7200 # wait at-most two hours before retrying
+    job_retry_limit: 5
```
Comment (sigurdm) [inline cron.yaml]: trailing newline

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Important changes to data-models, configuration and migrations between each
+version, listed here to easy deployment and troubleshooting.
```
Comment (sigurdm) [inline CHANGELOG.md]: easy -> ease

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Next Release
+
+ * Run `gcloud app deploy cron.yaml` to update cron-job retry logic.
+
+## 20190306t115839
+
```
Comment (sigurdm) [inline CHANGELOG.md]: You could add runtime-version of the deployment here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Important changes to data-models, configuration and migrations between each
+version, listed here to easy deployment and troubleshooting.
+
+## Next Release
+
+ * Run `gcloud app deploy cron.yaml` to update cron-job retry logic.
+
+## 20190306t115839
```
Comment (sigurdm) [inline CHANGELOG.md]: Prefix the id with something like `appengine id:`

Comment (jonasfj) [reply CHANGELOG.md]: Changed it to use git tag...

File: CHANGELOG.md (hunk 3 above)
Comment (jonasfj) [reply CHANGELOG.md]: ++

**Pull request merged**